net/datacenter-node-install/setup-validator.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
What is this script doing?  Looks like we call all those scripts already, and then randomize the password?  Why?

Reply:
@TristanDebrunner would have the best answer here, but I believe this is a separate workflow (for setting up nodes we send to validators) than what we used to setup the nodes destined for our colo rack.